walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -90,4 +79,46 @@
 
     return times;
   }
+
+  public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
```

Comment:
I know that this function didn't have documentation before but that was when it was the only function in the entire class. Now that the class has two functions, we might want to consider moving some off the documentation off the class level and adding it to the specific functions instead.

Reply:
I moved the description to the function instead of the class, does that work?